packages/grafana-data/src/datetime/formatter.ts

```diff
@@ -5,28 +5,89 @@ import { DateTimeInput } from './moment_wrapper';
 import { DEFAULT_DATE_TIME_FORMAT, MS_DATE_TIME_FORMAT } from './formats';
 import { DateTimeOptions, getTimeZone } from './common';
 
+/**
+ * Type describing the formatting options that can be passed to the {@link dateTimeFormat}
+ * helper function to control how the date and time value passed to the function will
+ * be formatted.
+ *
+ * @public
+ */
 export interface DateTimeOptionsWithFormat extends DateTimeOptions {
+  /**
+   * Specify a {@link https://momentjs.com/docs/#/displaying/format | momentjs} format to
+   * use a custom formatting pattern of the date and time value. If no format is set
+   * the {@link DEFAULT_DATE_TIME_FORMAT} will be used.
```

Comment:
```suggestion
   * then the {@link DEFAULT_DATE_TIME_FORMAT} is used.
```

Reply:
Whenever there is an "if", add a "then" in the sentence. This makes it clearer for non-native English speakers and noobs like me.